lib/puppet/type/service.rb

```diff
@@ -45,6 +45,8 @@ module Puppet
     feature :maskable, "The provider can 'mask' the service.",
       :methods => [:mask]
 
+    feature :configurable_timeout, "The provider can specify a timeout for service operations"
```

Comment:
This should also mention something about the `ensure` property in the feature description.

Reply:
Discussed in hipchat:

Since the feature is type (and not provider) specific it's possible future providers will use timeout in other ways, so we are going to leave out mentioning ensure